src/main/java/factionsplusplus/models/InteractionContext.java

```diff
@@ -1,13 +1,17 @@
 package factionsplusplus.models;
 
-import java.lang.annotation.Target;
 import java.util.UUID;
 
+import com.google.inject.Inject;
+
+import factionsplusplus.services.LocaleService;
+
 public class InteractionContext {
     private Type type;
     private TargetType targetType = null;
     private Gate gate = null;
     private UUID uuid = null;
+    @Inject private LocaleService localeService;
```

Comment:
This doesn't seem to be injecting correctly, keeps returning null. I saw your use of this in `CommandContext`, unsure what the issue could be with doing it here

Reply:
InteractionContext isn't created through guice so inject wouldn't work here -- i'm starting to wonder if using guice was a bad idea and we should explore the static interface method of doing DI. not sure.

we could make it be created using guice but would need some refactoring (whenever we want an interactioncontext we'd need to use a guice factory and have access to our FactionsPlusPlus instance)